x/stake/handler_test.go

```diff
@@ -372,6 +403,56 @@ func TestMultipleMsgCreateValidator(t *testing.T) {
 	}
 }
 
+func TestMultipleMsgCreateValidatorOnBehalfOf(t *testing.T) {
```

Comment:
I don't really think this test adds anything useful to comprehension of the code... doubt it will find any bugs... remove?

Reply:
Yea I think it might make sense to remove the above test and rename this, since this tests the more general case